### ISSUE TITLE
Bug: always crashes

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [jignesh13/googlecalendar#1]
Tested on Pixel 2 with Q beta 4, and Google Calendar is installed.
I always get this crash:

```
2019-06-29 21:38:37.425 24363-24363/? E/.GoogleCalenda: Unknown bits set in runtime_flags: 0x8000
2019-06-29 21:38:38.209 24363-24363/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.example.GoogleCalendar, PID: 24363
    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.example.GoogleCalendar/com.example.GoogleCalendar.MainActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.startsWith(java.lang.String)' on a null object reference
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3260)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3396)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2009)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7319)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:934)
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.startsWith(java.lang.String)' on a null object reference
        at com.example.GoogleCalendar.GooglecalenderView.init(GooglecalenderView.java:252)
        at com.example.GoogleCalendar.MainActivity.onCreate(MainActivity.java:166)
        at android.app.Activity.performCreate(Activity.java:7783)
        at android.app.Activity.performCreate(Activity.java:7772)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1299)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3235)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3396) 
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83) 
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2009) 
        at android.os.Handler.dispatchMessage(Handler.java:107) 
        at android.os.Looper.loop(Looper.java:214) 
        at android.app.ActivityThread.main(ActivityThread.java:7319) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:934) 

```

**Comment by jignesh13** [comment]
sorry let answer but i working on it i update soon thanks for your response

**Comment by jignesh13** [comment]
sorry for late answer pls download latest version of code now it work fine

**Comment by AndroidDeveloperLB** [comment]
Seems to work fine now.
But it's only one way to show the calendar?
And how did you make it so similar?

**Comment by jignesh13** [comment]
please check latest commit d83d61b
recently I add Schedule, Day View, MonthView, WeekView, 3-DayView same like google calendar app also you can easily switch between them and add many extra feature also.

**Comment by AndroidDeveloperLB** [comment]
How did you make this library to be so similar?

**Comment by jignesh13** [comment]
I always want to make app like google calendar app
below library inspire me for do that
[AgendaCalendarView](https://github.com/Tibolte/AgendaCalendarView),[scrollparallaximageview](https://github.com/gjiazhe/ScrollParallaxImageView),[Android-Week-View](https://github.com/alamkanak/Android-Week-View)
and i spend many day and night for make so similar

**Comment by AndroidDeveloperLB** [comment]
Seems it still crashes (tested on Pixel 4 Android S beta 2) :

[2021-06-20_13-30-25.zip](https://github.com/jignesh13/googlecalendar/files/6682121/2021-06-20_13-30-25.zip) 

[googlecalendar.zip](https://github.com/jignesh13/googlecalendar/files/6682120/googlecalendar.zip)


    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.example.GoogleCalendar/com.example.GoogleCalendar.MainActivity}: java.lang.NumberFormatException: null
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3608)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3765)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:102)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2193)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loopOnce(Looper.java:201)
        at android.os.Looper.loop(Looper.java:288)
        at android.app.ActivityThread.main(ActivityThread.java:7796)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:974)
     Caused by: java.lang.NumberFormatException: null
        at java.lang.Long.parseLong(Long.java:557)
        at java.lang.Long.parseLong(Long.java:636)
        at com.example.GoogleCalendar.Utility.readCalendarEvent(Utility.java:141)
        at com.example.GoogleCalendar.MainActivity.onCreate(MainActivity.java:627)
        at android.app.Activity.performCreate(Activity.java:8036)
        at android.app.Activity.performCreate(Activity.java:8016)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1329)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3581)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3765) 
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:102) 
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2193) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loopOnce(Looper.java:201) 
        at android.os.Looper.loop(Looper.java:288) 
        at android.app.ActivityThread.main(ActivityThread.java:7796) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:974)

**Comment by jignesh13** [comment]
issue is solved now please check commit deb8e31
